docs/content/docs/connectors/datastream/formats/parquet.md

```diff
@@ -0,0 +1,99 @@
+---
+title:  "Parquet files"
+weight: 4
+type: docs
+aliases:
+- /dev/connectors/formats/parquet.html
+- /apis/streaming/connectors/formats/parquet.html
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+
+# Parquet format
+
+Flink supports reading [parquet](https://parquet.apache.org/) files and producing [Flink rows](https://nightlies.apache.org/flink/flink-docs-master/api/java/org/apache/flink/types/Row.html).
+To use that format you need to add the dependency below to your project:
+
+```xml
+{{< artifact flink-parquet >}}
+```  
+
+The DataStream API is now compatible with both batch and streaming execution modes. 
+And this format is compatible with the new Source that can be used in both modes.
+Thus, you can use this format in two ways:
+- Bounded read for batch mode
+- Continuous read for streaming mode: monitors a directory for new files that appear 
```

Comment:
```suggestion
After adding the dependency, you can read and write Parquet data in either the [Batch Execution Mode or Streaming Execution Mode]({{< ref "/docs/dev/datastream/execution_mode" >}}).
```

Reply:
I think it is important for the user to know that for file reading in streaming mode a monitoring of the directory will be in place and the referred doc does not mention this.